src/App.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.css';
 
 export const App = () => (
```

Comment:
use only one style to export components
```suggestion
const App = () => (
```